index.html

```diff
@@ -2757,17 +2779,31 @@ <h3>Algorithm</h3>
                       either <code>@language</code>, <code>@index</code>, <code>@id</code>, or <code>@type</code>
                       based on the contents of <var>container</var>, as <var>var</var>, and <code>true</code>
                       for <var>vocab</var>.</li>
+                    <li class="changed">Set <var>index key</var> to the value of <a>index mapping</a> in
+                      the <a>term definition</a> associated with <var>item active property</var> in <var>active context</var>,
+                      or <code>@index</code>, if no such value exists.</li>
                     <li>If <var>container</var> includes <code>@language</code> and
                       <var>expanded item</var> contains a
                       <code>@value</code> <a>member</a>, then set <var>compacted item</var>
                       to the value associated with its <code>@value</code> <a>member</a>.
                       Set <var>map key</var> to the value of <code>@language</code> in <var>expanded item</var>, if any.</li>
-                    <li>If <var>container</var> includes <code>@index</code> set <var>map key</var> to the value of <code>@index</code> in <var>expanded item</var>, if any,
+                    <li>Otherwise, if <var>container</var> includes <code>@index</code>
+                      and <var>index key</var> is <code>@index</code>,
+                      set <var>map key</var> to the value of <code>@index</code> in <var>expanded item</var>, if any,
                       and remove <var>container key</var> from <var>compacted item</var>.</li>
-                    <li class="changed">If <var>container</var> includes <code>@id</code>, set
+                    <li class="changed">Otherwise, if <var>container</var> includes <code>@index</code>
+                      and <var>index key</var> is not <code>@index</code>,
+                      Set <var>map key</var> to the first value of <var>container key</var> in <var>compacted item</var>, if any.
```

Comment:
shouldn't it be `<var>index key</var>` instead of `<var>container key</var>`?

Reply:
No, this is a bit tricky, as the index case is handled differently from other types of indexing earlier in the compaction algorithm. This is one of the things that make index maps different from other maps. If we were to re-write the compaction algorithm from scratch, we might do it differently.